Client/Frontend/Browser/TabsBar/TabsBarViewController.swift

```diff
@@ -53,8 +53,12 @@ class TabsBarViewController: UIViewController {
     fileprivate weak var tabManager: TabManager?
     fileprivate var tabList = WeakList<Tab>()
     
+    private var activeTheme: Theme
+    
     init(tabManager: TabManager) {
         self.tabManager = tabManager
+        activeTheme = Theme.of(tabManager.selectedTab)
+        
```

Comment:
Instead of tracking a theme, perhaps we can follow what was done in TabTrayController: 

https://github.com/brave/brave-ios/blob/613103e8fdf98cb85e159546476be10baec5ca8b/Client/Frontend/Browser/TabTrayController.swift#L415-L424

`styleChildren` is a default method of the `Themable` protocol. Then we just call `cell.applyTheme(Theme.of(tab))`

Reply:
Defined the themeableChildren for the viewcontroller to update cell theme.